docs/blog/2018-10-11-rise-of-modern-web-development/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Journey to the Content Mesh Part 3: The Rise of Modern Web Development"
+author: "Sam Bhagwat"
+"date": 2018-10-09
+tags: ["content-mesh"]
+---
+
+_This is Part 3 of a series. Part 1 is_ [The Journey to a Content Mesh](/blog/2018-10-04-journey-to-the-content-mesh) _and Part 2 is_ [Unbundling of the CMS](/blog/2018-10-10-unbundling-of-the-cms).
+
+Developing for the web is _hard_. With millions of properties on the Internet, website developers have to create experiences that are compelling enough to engage audiences when new destinations are only a few keystrokes away.
+
+And while creating rich user experiences is never an easy task, standard content web development environments introduce additional challenges. Teams are forced to work around them, but ultimately deliver less rich experiences at higher cost.
+
+## Traditional CMS development presents challenges
+
+Some challenges of traditional CMS development environments include:
+
+- **Walled-garden development.** Work on projects across CMS systems often requires reimplement basic functionality like carousels or banners. Development can be blocked due to CMS access restrictions or code freezes. Upgrade paths can be challenging when CMSs don’t support component UI versioning.
+
+- **Maintaining local environments**. Setting up a local app server and database and keeping it up to date with team members’ changes is time-consuming, especially when switching between projects or returning after a gap in development.
+
+- **Project organization.** Reliable installing and managing third-party dependencies, including cross-compatibility and handling bugs in upstream, is challenging. Vanilla JavaScript offers no standard project organization or code bundling patterns.
```

Comment:
Suggested tweak:

"Reliable installation and management of third-party dependencies..."

or

"Reliably installing and managing third-party dependencies..."

Reply:
👍